From: Richard Fröhning <misanthropos@gmx.de>
Date: Tue, 23 Apr 2024 19:56:10 +0200
Subject: dts file for wr841-mod16/64

diff --git a/target/linux/ath79/dts/qca9533_tplink_tl-wr841-rom16ram64-v10.dts b/target/linux/ath79/dts/qca9533_tplink_tl-wr841-rom16ram64-v10.dts
new file mode 100644
index 0000000000000000000000000000000000000000..80b828abd6015b4ef060c186bb5496028fce155f
--- /dev/null
+++ b/target/linux/ath79/dts/qca9533_tplink_tl-wr841-rom16ram64-v10.dts
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_tplink_tl-wr841.dtsi"
+
+/ {
+	compatible = "tplink,tl-wr841-rom16ram64-v10", "qca,qca9533";
+	model = "TP-Link TL-WR841N/ND MOD16/64 v10";
+
+	aliases {
+		led-boot = &led_qss;
+		led-failsafe = &led_qss;
+		led-running = &led_qss;
+		led-upgrade = &led_qss;
+	};
+};
