From: Richard Fröhning <misanthropos@gmx.de>
Date: Thu, 22 Aug 2024 05:52:33 +0200
Subject: add qca9533_tplink_tl-wr841n-nd-mod8-64-v10.dts

diff --git a/target/linux/ath79/dts/qca9533_tplink_tl-wr841n-nd-mod8-64-v10.dts b/target/linux/ath79/dts/qca9533_tplink_tl-wr841n-nd-mod8-64-v10.dts
new file mode 100644
index 0000000000000000000000000000000000000000..edbccddd4bf5e501bd8ffc13eaea9be1da821223
--- /dev/null
+++ b/target/linux/ath79/dts/qca9533_tplink_tl-wr841n-nd-mod8-64-v10.dts
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_tplink_tl-wr841-rom8ram64.dtsi"
+
+/ {
+	compatible = "tplink,tl-wr841n-nd-mod8-64-v10", "qca,qca9533";
+	model = "TP-Link TL-WR841N/ND MOD8/64 v10";
+
+	aliases {
+		led-boot = &led_qss;
+		led-failsafe = &led_qss;
+		led-running = &led_qss;
+		led-upgrade = &led_qss;
+	};
+};
